README.md

```diff
@@ -1,19 +1,20 @@
 # Brigade Windows Go Tools Image
 [![slack](https://img.shields.io/badge/slack-brigade-brightgreen.svg?logo=slack)](https://kubernetes.slack.com/messages/C87MF1RFD)
 
-This project provides a curated set of tools to facilitate building
-[Brigade](https://github.com/brigadecore/brigade) components for _Windows_
-containers. Currently, that amounts to the base
-`golang:<version>-windowsservercore-1809`, plus `cmake` and
-[MinGW](https://www.mingw-w64.org/installed) installed via
-[Chocolately](https://chocolatey.org/).
+This project provides a curated set of tools to facilitate building Go-based
+components of [Brigade](https://github.com/brigadecore/brigade) (even those
+requiring CGO) for _Windows_ containers. Currently, that amounts to the standard
+Go toolchain, [MinGW](https://www.mingw-w64.org/installed), and `cmake`, all
```

Comment:
I see a 404 on this MinGW link... is there a different URL that can be used?

Reply:
Hmm... not sure what happened there. Will fix.